src/v2/Apps/Conversation/Components/__tests__/Conversation.jest.tsx

```diff
@@ -64,9 +64,19 @@ describe("Conversation", () => {
       }
     })
 
-    it("shows the buyer guarantee message", async () => {
+    it("shows the buyer guarantee message if offerable", async () => {
       renderWithRelay({
-        Artwork: () => ({ isOfferable: true }),
+        Artwork: () => ({ isOfferable: true, isOfferableFromInquiry: false }),
+      })
+
+      expect(
+        screen.getByText("Be Protected by The Artsy Guarantee")
+      ).toBeInTheDocument()
+    })
+
+    it("shows the buyer guarantee message if offerable from inquiry", async () => {
+      renderWithRelay({
+        Artwork: () => ({ isOfferableFromInquiry: true, isOfferable: false }),
```

Comment:
Is there a test with both `isOfferable` and `isOfferableFromInquiry` `true`?

Reply:
It's an `||` so I didn't think it was needed, but can quickly add it to be foolproof 👍

Reply:
Added. Resolving this 📕